Add a prereq section on what to know before doing Scan 2.0 (#3018)
pivotal/docs-tap: scst-scan/bring-your-own-scanner.hbs.md

1. [Create an ImageVulnerabilityScan](ivs-create-your-own.hbs.md): Create an ImageVulnerabilityScan template that tells the Tanzu Application Platform how to run your scanner.
2. [Verify your ImageVulnerabilityScan](verify-app-scanning.hbs.md): Verify that your ImageVulnerabilityScan is working correctly so that downstream Tanzu Application Platform services work correctly.
3. [Wrap your ImageVulnerabilityScan in a ClusterImageTemplate](clusterimagetemplates.hbs.md): The ClusterImageTemplate wraps the ImageVulnerabilityScan and allows the Tanzu Application Platform supply chain to run the scan job.

## <a id="prerequisities"></a>Prerequisites

TAP users will need to:

- Provide their own Vulnerability Scanner Image either by:
  - Using a publicly available image that contains the scanner CLI.

>     - For example, the official image from Aqua for Trivy from [Dockerhub](https://hub.docker.com/r/aquasec/trivy/tags).

Reviewer: ```suggestion
    - For example, the official Aqua image for Trivy from [Dockerhub](https://hub.docker.com/r/aquasec/trivy/tags).
```

1. [Create an ImageVulnerabilityScan](ivs-create-your-own.hbs.md): Create an ImageVulnerabilityScan template that tells the Tanzu Application Platform how to run your scanner.
2. [Verify your ImageVulnerabilityScan](verify-app-scanning.hbs.md): Verify that your ImageVulnerabilityScan is working correctly so that downstream Tanzu Application Platform services work correctly.
3. [Wrap your ImageVulnerabilityScan in a ClusterImageTemplate](clusterimagetemplates.hbs.md): The ClusterImageTemplate wraps the ImageVulnerabilityScan and allows the Tanzu Application Platform supply chain to run the scan job.

## <a id="prerequisities"></a>Prerequisites

TAP users will need to:

- Provide their own Vulnerability Scanner Image either by:

>   - Using a publicly available image with the scanner CLI.

Reviewer: ```suggestion
  - Using a publicly available image that contains the scanner CLI.
```

>     - For example, this image with Trivy scanner CLI from [Dockerhub](https://hub.docker.com/r/aquasec/trivy/tags).

Reviewer: ```suggestion
    - For example, the official image from Aqua for Trivy from [Dockerhub](https://hub.docker.com/r/aquasec/trivy/tags).
```

  - Building their own image with the scanner CLI which allows for:
    - More customizable scanning experience.
      - For example, creating an image with the scanner CLI along with any required dependencies to run the scanner CLI.

>     - Managing vulnerabilities within the TAP user's custom image.

Reviewer: Can you elaborate on this a bit?  I'm not sure what you are referring to.  Does this mean patching vulnerabilities that exist in it?

Author: Yes I meant patching vulnerabilities that existing the image which was an issue for us in Scan 1.0. But if the TAP user's build their own image, they would have more control on addressing the vulnerabilities that exist in it.

Author: I removed this line as requested.

>     - Managing their own image to meet the TAP user's legal compliance standards.

Reviewer: ```suggestion
    - Managing their own image to meet the TAP user's compliance standards.
```
I like this, but I'd just say compliance standards.  I don't think we need to bring legal in to it :)
